Research/Research/DataModel/Objects/Step/Question/JsonChoiceObject.swift

```diff
@@ -44,7 +44,7 @@ public extension JsonComparable {
     }
 }
 
-public protocol JsonChoice : RSDChoice, JsonComparable, Codable {
+public protocol JsonChoice : RSDChoice, JsonComparable { //, Codable {
```

Comment:
does leaving this comment here serve a purpose?

Reply:
no, mistake. thanks!